src/components/popup/Route.tsx

```diff
@@ -26,10 +26,9 @@ export const Wrapper = styled(motion.div)<{
   expanded?: boolean;
 }>`
   position: relative;
-  width: ${(props) => (props.responsive ? "100%" : "377.5px")};
+  width: ${(props) => (props.responsive ? "100%" : "377px")};
   min-height: ${(props) => (props.expanded ? "100vh" : "600px")};
   max-height: max-content;
-  background-color: rgb(${(props) => props.theme.background});
```

Comment:
The background is set on the `body` by the `ThemeProvider` in the components library, so no need to set it again here.